meta-ostro/recipes-swupd/swupd-client/oe-swupd-helpers.bbappend

```diff
@@ -1,3 +0,0 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
-
-SRC_URI_append = "file://efi-combo-trigger.service"
```

Comment:
On Mon, 2016-04-04 at 04:46 -0700, Dmitry Rozhkov wrote:
> In meta-ostro/recipes-swupd/swupd-client/oe-swupd-helpers.bbappend:
> 
> > @@ -1,3 +0,0 @@
> > -FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
> > -
> > -SRC_URI_append = "file://efi-combo-trigger.service"
> 
> Previously meta-swupd's oe-swupd-helpers.bb was responsible for
> installing the file efi-combo-trigger.service. Now nothings install it
> onto the target. As result update-triggers.service doesn't actually
> run efi-combo-updater.

Fix is in commit 8c8fcbd6b69bec: installing and enabling the service has
to be done explicitly, which also depended on further changes to get rid
of the undesired update-check.

Please re-test.